src/components/anchorButton/AnchorButton.tsx

```diff
@@ -0,0 +1,29 @@
+type Icon = {
+  leftIcon?: JSX.Element
+  rightIcon?: JSX.Element
+}
+
+type AnchorButtonProps = {
+  url?: string
+  name: string
+  icon?: Icon
+}
+
+const AnchorButton: React.FC<AnchorButtonProps> = ({ url, name, icon }): JSX.Element => (
```

Comment:
I feel like we can't understand each other.

What I want to say is, despite the fact that a `a` tag without an `href` attribute seems to be valid HTML5, it makes no sense, whatever you call it `Link` or `AnchorButton`. And the same for a `Button` that takes an `url` property and returns an anchor.

So, what I purpose, is to create an `Anchor.tsx` file that will contain both `NamedAnchor` and `LinkedAnchor` components that share the same structure in terms of properties, rendering and style from a base component.

Then, if the page, you could render either a `NamedAnchor` or a `LinkedAnchor` if an url is available.

This concern is mainly about HTML semantic.